packages/calypso-build/package.json

```diff
@@ -37,13 +37,13 @@
 		"@automattic/mini-css-extract-plugin-with-rtl": "^0.8.0",
 		"@babel/cli": "^7.8.4",
 		"@babel/core": "^7.8.4",
+		"@babel/helpers": "^7.9.0",
```

Comment:
All I did was update terser-webpack-plugin, not sure why that re-ordered some random items in this file. 🤷